src/features/ITGovernance/Admin/Dates/index.tsx

```diff
@@ -1,277 +1,382 @@
+          grbReviewType
         }
       }
-    }).then(() => {
-      history.push(`/it-governance/${systemId}/intake-request`);
     });
+
+    if (grbReviewType === SystemIntakeGRBReviewType.ASYNC) {
+      await updateType;
+      history.push(`/it-governance/${systemId}/intake-request`);
+    } else {
+      await updateType;
+
+      const newGrtDate = DateTime.fromObject(
+        {
+          day: Number(grtDateDay),
+          month: Number(grtDateMonth),
+          year: Number(grtDateYear)
+        },
+        { zone: 'UTC' }
+      ).toISO();
+
+      const newGrbDate = DateTime.fromObject(
+        {
+          day: Number(grbDateDay),
+          month: Number(grbDateMonth),
+          year: Number(grbDateYear)
+        },
+        { zone: 'UTC' }
+      ).toISO();
+
+      await updateReviewDates({
+        variables: {
+          input: {
+            id: systemId,
+            grtDate: newGrtDate,
+            grbDate: newGrbDate
+          }
+        }
+      });
+
+      history.push(`/it-governance/${systemId}/intake-request`);
+    }
   };
 
+  /**
+   * Returns the correct status data for the review type,
+   * or NOT_STARTED if status is null
+   */
+  const grbReviewStatus: GRBReviewStatus | null | undefined = useMemo(() => {
+    if (!grbReviewStartedAt) return 'NOT_STARTED';
+
+    return systemIntake.grbReviewType === SystemIntakeGRBReviewType.STANDARD
+      ? grbReviewStandardStatus
+      : grbReviewAsyncStatus;
+  }, [
+    grbReviewAsyncStatus,
+    grbReviewStandardStatus,
+    grbReviewStartedAt,
+    systemIntake.grbReviewType
+  ]);
+
+  const reviewIsInProgress: boolean =
+    grbReviewStatus !== 'NOT_STARTED' && grbReviewStatus !== 'COMPLETED';
+
+  if (!grbReviewStatus) {
+    return null;
+  }
+
   return (
-    <Formik
-      initialValues={initialValues}
-      onSubmit={onSubmit}
-      validationSchema={DateValidationSchema}
-      validateOnBlur={false}
-      validateOnChange={false}
-      validateOnMount={false}
-    >
-      {(formikProps: FormikProps<SubmitDatesForm>) => {
-        const { errors, setErrors, handleSubmit } = formikProps;
-        const flatErrors = flattenErrors(errors);
-        return (
-          <>
-            {Object.keys(errors).length > 0 && (
-              <ErrorAlert
-                testId="system-intake-errors"
-                classNames="margin-top-3"
-                heading={t('form:inputError.checkFix')}
-              >
-                {Object.keys(flatErrors).map(key => {
-                  return (
-                    <ErrorAlertMessage
-                      key={`Error.${key}`}
-                      errorKey={key}
-                      message={flatErrors[key]}
+    <>
+      <PageHeading data-testid="grt-dates-view" className="margin-top-0">
+        {t('governanceReviewTeam:dates.heading')}
+      </PageHeading>
+      <h2>{t('governanceReviewTeam:dates.subheading')}</h2>
+      <Form onSubmit={handleSubmit(onSubmit)} className="maxw-mobile-lg">
```

Comment:
The form width looks a little off compared to the Figma. It looks like adding the `grid-col-8` util class here works instead of setting the max width.